presto-redshift/src/main/java/io/prestosql/plugin/redshift/RedshiftClient.java

```diff
@@ -62,6 +72,76 @@ protected void renameTable(JdbcIdentity identity, String catalogName, String sch
         }
     }
 
+    static String generateCreateTableSql(String fullyQualifiedTableName, ImmutableList.Builder<String> columnList, Optional<String> distKey, Optional<String> compoundSortKeys, Optional<String> interleavedSortKeys)
```

Comment:
mark it with `@VisibleForTesting`
Please do not use output parameters like `ImmutableList.Builder<String> columnList`. You might want to return something more complex than `String`